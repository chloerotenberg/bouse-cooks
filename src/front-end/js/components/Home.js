import React from 'react';

export default class Home extends React.Component {
  render() {
    const names = [
      'booty',
      'best',
      'bokbok',
      'best butts',
      'bee',
      'blue',
      'bonfire',
      'big ups to all my haters',
      'bird',
      'big'
    ];
    const currName = names[Math.floor(Math.random() * names.length)];

    return (
      <div className="content home">
        <div className="centered">{currName} house</div>
      </div>
    );
  }
}
