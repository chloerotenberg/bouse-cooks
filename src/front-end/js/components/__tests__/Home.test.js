import React from 'react';
import { shallow } from 'enzyme';

import Home from '../Home';

describe('Home component', () => {
  it('matches snapshot', () => {
    global.Math.random = () => 0;

    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});
