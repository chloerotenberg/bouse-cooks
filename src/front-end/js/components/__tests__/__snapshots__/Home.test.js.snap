// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component matches snapshot 1`] = `
<div
  className="content home"
>
  <div
    className="centered"
  >
    booty
     house
  </div>
</div>
`;
